client/my-sites/themes/collections/theme-collections-layout.tsx

```diff
@@ -6,23 +6,23 @@ export interface ThemeCollectionsLayoutProps {
 	getActionLabel: ( themeId: string ) => string;
 	getOptions: ( themeId: string ) => void;
 	getScreenshotUrl: ( themeId: string ) => string;
-	onTierSelect: ( tier: string ) => void;
+	onSeeAll: ( filters: object ) => void;
```

Comment:
The reason for this been called onTierSelect and not onSeeAll is that once we implement the rest of the carousels we're going to need to pass multiple references to specific parts of the theme showcase component. For example, onTierSelect and onFilterSelect, and then in this file we need to supply the correct dependency to the concrete ShowcaseThemeCollection instances so that they can operate the ThemeShowcase.

Let's please revert to the original naming.